Resources/Prototypes/_NF/Entities/Structures/Machines/Computers/computers_shipyard.yml

```diff
@@ -44,9 +44,6 @@
 - type: entity
   parent: ComputerShipyardBase
   id: ComputerShipyard
-  components:
-  - type: ShipyardConsole
-    newJobTitle: Captain
```

Comment:
Don't really agree with this.  Mercenaries, pilots and contractors should receive a new title, as it:
* allows ease of inspection for who's in charge larger ships
* provides an easy sense of accomplishment or distinguishment
* shows at a glance that somebody has, in fact, purchased a ship

I don't think _station jobs_ should receive the title, but that's another matter.

Reply:
It looks like all of the shipyards need to have their parenting heirarchy examined because its being inherited by SR/pirate/syndicate consoles and others and over-riding more appropriate job titles. I agree that civillian roles purchasing ships should still gain the 'captain' title, it has no gameplay impact on frontier other than social/status and RP.

Reply:
I specifically changed things so the "normal" shipyard console (ComputerShipyard) is separate from ComputerShipyardBase, the latter of which is used for non-civilian things. 🤔 but if we want the cap title back, I'll undo this one change and it should just work as before.

Reply:
It has been undone.